teleport-server/r53.tf

```diff
@@ -5,3 +5,11 @@ resource "aws_route53_record" "teleport" {
   records = [aws_eip.teleport_public.public_dns] # This way it resolves to the instance private IP within the VPC
   ttl     = "300"
 }
+
+resource "aws_route53_record" "teleport_sub" {
+  zone_id = data.aws_route53_zone.root.zone_id
+  name    = "*.${local.teleport_domain_name}"
+  type    = "CNAME"
+  records = [aws_eip.teleport_public.public_dns] # This way it resolves to the instance private IP within the VPC
```

Comment:
you say private IP but you use the public IP?

Reply:
An EIP public_dns auto-resolves to a private IP when queried within VPC

Reply:
ah right indeed. AWS magic